index.html

```diff
@@ -239,27 +243,27 @@ <h3>Skills</h3>
           </li>
           <li>
             <a href="#">
-              <img src="Icons/Linkedin-icon.svg" alt="twitter">
+              <img src="Icons/Linkedin-icon.svg" alt="linkedin">
             </a>
           </li>
           <li>
             <a href="#">
-              <img src="Icons/medium.svg" alt="twitter">
+              <img src="Icons/medium.svg" alt="medium">
             </a>
           </li>
           <li>
             <a href="#">
-              <img src="Icons/github.svg" alt="twitter">
+              <img src="Icons/github.svg" alt="github">
             </a>
           </li>
           <li>
             <a href="#">
-              <img src="Icons/angellist.svg" alt="twitter">
+              <img src="Icons/angellist.svg" alt="angellist">
             </a>
           </li>
         </ul>
       </div>
-      <img class="ft-indicator" src="./Icons/Shape.svg" alt="shape">
+      <img class="ft-indicator" src="./Icons/Shape.svg" alt="">
```

Comment:
- [ ] Good job working on the accessibility 👍, but still need to to fix some points 😃. Kindly, make sure every image has `alt` with appropriate alternative text, that's useful for interacting or understanding the page has an `alt` text
- [ ] Also the line [76](https://github.com/Yakub-Ojo/My-Portfolio/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L75:~:text=%3Cimg%20src%3D%22./Icons/Indicator.svg%22%20alt%3D%22%22%20/%3E) `<img src="./Icons/Indicator.svg" alt="" />` . Kindly, make sure every image has `alt` with appropriate alternative text, that's useful for interacting or understanding the page has an `alt` text 👍